client/src/pages/TakeSurvey/index.jsx

```diff
@@ -15,6 +16,7 @@ const TakeSurvey = () => {
   useEffect(() => {
     const getSurvey = async () => {
       const response = await axios.get('/surveys/508f1f99bcf86cd799439014');
```

Comment:
Confused why there is a hardcoded hash after `surveys/`? The link looks to be targeting a specific survey. If it's meant to be dynamic then this looks like it will take the same survey every time!!

Might be worth raising as an issue on the repo for now.

Reply:
this was a fix until we made the links for each participant, I can probs change it now